meta-luneos/recipes-luneos/apps/org.webosports.app.settings-qml.bb

```diff
@@ -22,6 +22,15 @@ WEBOS_REPO_NAME = "org.webosports.app.settings"
 SRC_URI = "${WEBOS_PORTS_GIT_REPO_COMPLETE}"
 S = "${WORKDIR}/git"
 
+# We don't provide android-property-service on non-Android devices, so remove it in these cases to avoid LS2 warnings/errors
+
+FOO:halium = ""
```

Comment:
can we use something more explicit than a generic "FOO" variable ?
For instance "REMOVE_ANDROID_PROPERTY_SERVICE_CMD" ?